contrib/core-contract-tests/tests/pox-4/pox-4.stateful-prop.test.ts

```diff
@@ -66,6 +66,9 @@ describe("PoX-4 invariant tests", () => {
         isStacking: false,
         hasDelegated: false,
         delegatedTo: "",
+        wasDelegated: false,
```

Comment:
Perhaps we need a better name here? It’s confusing to have both a hasDelegated and a wasDelegated field.

What’s the purpose served by each field?

Reply:
Yep, I was thinking about: 
- `hasDelegated` is true if the wallet has delegated another wallet.
- `wasDelegated` is true if the wallet was delegated by another wallet

We can change it to something like `gaveDelegation` and `wasDelegated`

Reply:
@friedger, what do you think about this naming pattern?